seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2647,6 +2647,48 @@
       }
     },
     {
+      "id": "cxk61uhxor7m4a1swj1ha9eo",
```

Comment:
@DusanSacha, this needs a real MongoDB ID, which you get by opening MongoDB in your terminal and running `ObjectId()`